src/middlewared/middlewared/etc_files/exports.mako

```diff
@@ -95,6 +96,15 @@
 
         return hostname
 
+    def disable_exportsd():
+        if middleware.call_sync('filesystem.is_immutable', '/etc/exports.d'):
```

Comment:
What if user removes immutable flag, puts his stuff there and sets it back? :grinning:

Reply:
Indeed. I was not thinking creatively enough.